packages/babel-plugin-transform-for-of/src/no-helper-implementation.js

```diff
@@ -0,0 +1,192 @@
+import { template, types as t } from "@babel/core";
+
+// This is the legacy implementation, which inlines all the code.
+// It must be kept for compatibility reasons.
+// TODO (Babel 8): Remove this code.
+
+export default function transformWithoutHelper(loose, path, state) {
```

Comment:
This file contains the old plugin code needed when the helper isn't available. I just copy-pasted it without any edit.